include/seqan3/argument_parser/detail/format_help.hpp

```diff
@@ -87,20 +87,23 @@ class format_help : public format_help_base<format_help>
         unsigned rightColumnTab;
 
         //!\brief The constructor.
-        console_layout_struct() :
+        //!\param[in] terminal_width The width of the terminal.
+        console_layout_struct(unsigned terminal_width) :
```

Comment:
```suggestion
        console_layout_struct(uint32_t const terminal_width) :
```

Reply:
The underlying type is `unsigned`, so either change all of the fields to `uint32_t` or use `unsigned` in the constructor.

Reply:
Yes change all of the fields. That's what I am implying 😏